packages/main/src/MultiInput.js

```diff
@@ -0,0 +1,188 @@
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Input from "./Input.js";
+import MultiInputTemplate from "./generated/templates/MultiInputTemplate.lit.js";
+import styles from "./generated/themes/MultiInput.css.js";
+import Token from "./Token.js";
+import Tokenizer from "./Tokenizer.js";
+import Icon from "./Icon.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-multi-input",
+	properties: /** @lends sap.ui.webcomponents.main.MultiInput.prototype */ {
+		/**
+		 * Determines whether a value help icon will be should in the end of the input.
+		 * Pressing the icon will fire <code>value-help-icon-press</code> event.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		showValueHelpIcon: {
+			type: Boolean,
+		},
+
+		/**
+		 * Indicates whether the tokenizer is expanded or collapsed(shows the n more label)
+		 * @private
+		 */
+		expandedTokenizer: {
+			type: Boolean,
+		},
+	},
+	slots: /** @lends  sap.ui.webcomponents.main.MultiInput.prototype */ {
+		/**
+		 * Defines the <code>ui5-multi-input</code> tokens.
+		 * <br><br>
+		 * Example: <br>
+		 * &lt;ui5-multi-input><br>
+		 * &nbsp;&nbsp;&nbsp;&nbsp;&lt;ui5-token slot="tokens" text="Token 1">&lt;/ui5-token><br>
+		 * &nbsp;&nbsp;&nbsp;&nbsp;&lt;ui5-token slot="tokens" text="Token 2">&lt;/ui5-token><br>
+		 * &lt;/ui5-multi-input>
+		 * <br> <br>
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		tokens: {
+			type: Token,
+			multiple: true,
```

Comment:
all slots are multiple, this is only for properties now, so I would suggest creating a default slot:
```js
default: {
  type: HTMLElement,
  propertyName: "tokens"
}
```

Reply:
Items is a default slot for input and its ancestors - MultiInput